src/Components/Main.js

```diff
@@ -3,19 +3,17 @@ import ToDoForm from './Todo/ToDoForm';
 import ToDoList from './Todo/ToDoList';
 import { Navbar } from 'react-bootstrap';
 import { Container, Row, Col } from 'react-bootstrap';
-import { useState } from 'react';
+import useFetch from '../Components/Hooks/useFetch';
+import Auth from './Auth';
 import './main.css';
+import useAuth from './Hooks/useAuth';
 
-const data = [
-  { title: "Do the dishes, clean kitchen", difficulty: 5, assignedTo: "Ian", completed: true },
-  { title: "Vaccuum and sweep all floors", difficulty: 3, assignedTo: "Sarah", completed: false },
-  { title: "Walk the dogs and pick up poop in yard", difficulty: 10, assignedTo: "Sarah", completed: false },
-  { title: "Wash and dry clothes. Make sure to fold them afterwards", difficulty: 7, assignedTo: "Ian", completed: true },
-];
+const toDoApi = 'https://deltav-todo.azurewebsites.net/api/v1/Todos';
 
 function Main() {
-
-  const [tasks, setTasks] = useState(data);
+  function setTasks() { }
+  const { tasks, reload } = useFetch(toDoApi);
```

Comment:
Need to remove this!

Two options:
1. Remove this function and fix all the places that use it. Use `reload()` instead.
2. Include `setTasks` in the `return` from your `useFetch`, so you can do an in-memory state update before you reload from the server. This is a more responsive UI, because we can show the toggle or new task or whatever without waiting on the server.
    ```suggestion
      const { tasks, reload, setTasks } = useFetch(toDoApi);
    ```